site/content/en/docs/tasks/run_python_jobs.md

```diff
@@ -277,3 +277,95 @@ hello world
 
 You can further customize the job, and can ask questions on the [Flux Operator issues board](https://github.com/flux-framework/flux-operator/issues).
 Finally, for instructions for how to do this with YAML outside of Python, see [Run A Flux MiniCluster](/docs/tasks/run_flux_minicluster/).
+
+### MPI Operator Job
+
+For this example, we will be using the [MPI Operator](https://www.kubeflow.org/docs/components/training/mpi/)
+to submit a job, and specifically using the [Python SDK](https://github.com/kubeflow/mpi-operator/tree/master/sdk/python/v2beta1) to do this easily. Given our Python environment created in the [setup](#before-you-begin), we can install this Python SDK directly to it as follows:
+
+```bash
+git clone --depth 1 https://github.com/kubeflow/mpi-operator /tmp/mpijob
+cd /tmp/mpijob/sdk/python/v2beta1
+python setup.py install
+cd -
+```
+
+Importantly, the MPI Operator *must be installed before Kueue* for this to work! Let's start from scratch with a new Kind cluster.
+We will also need to [install the MPI operator](https://github.com/kubeflow/mpi-operator/tree/master#installation) and Kueue. Here we install
+the exact versions tested with this example:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kubeflow/mpi-operator/master/deploy/v2beta1/mpi-operator.yaml
+kubectl apply -f https://github.com/kubernetes-sigs/kueue/releases/download/v0.4.0/manifests.yaml
+```
+You might need to pause until Kueue is ready, and then:
```

Comment:
We can be more direct here.

We can use kubectl with --wait flag maybe?

Reply:
hmm. In that case we just tell them they have to wait until everything is running. Using might in this case may cause confusion. 

```
You need to wait until Kueue is ready.
```

And maybe we ask them to do a `kubectl get po -n kueue-system` to confirm it all running.

Reply:
okay all set!